hera_mc/cm_utils.py

```diff
@@ -126,9 +120,42 @@ def add_date_time_args(parser):
         '--time', help="UTC hh:mm or float (hours), must include --date if use --time", default=0.0)
 
 
+def is_active(at_date, start_date, stop_date):
+    stop_date = get_stopdate(stop_date)
+    return at_date >= start_date and at_date <= stop_date
+
+
+def future_date():
+    """
+    Future is defined here, since defining a far FUTURE_DATE typically gives a
+    warning about UTC vs UT1 etc
+    """
+    return Time.now() + TimeDelta(300, format='jd')
+
+
+def get_stopdate(stop_date):
+    if isinstance(stop_date, Time):
+        return stop_date
+    else:
+        return future_date()
```

Comment:
This chain of logic should be pickier, in case someone accidentally passes a wrong argument to the function (or to `is_active` when they are merged). Assuming that the "default" case corresponds to an argument of None:

```python
if isinstance(stop_date, Time):
  return stop_date
if stop_date is None:
  return future_date()
raise ValueError('unexpected value for "stop_date" argument: %r' % (stop_date,))
```

(The parenthesization in the last line makes sure that the string formatting works reliably even if `stop_date` happens to have been passed in as a tuple.)